ocrd_webapi/routers/user.py

```diff
@@ -0,0 +1,64 @@
+"""
+module for implementing the authentication section of the api
+"""
+import logging
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.security import HTTPBasic, HTTPBasicCredentials
+
+from ocrd_webapi.authentication import authenticate_user, register_user
+
+router = APIRouter(
+    tags=["User"],
+)
+
+logger = logging.getLogger(__name__)
+# TODO: This may not be ideal, discussion needed
+security = HTTPBasic()
+
+
+@router.get("/user/login")
+async def user_login(auth: HTTPBasicCredentials = Depends(security)):
+    email = auth.username
+    password = auth.password
+    if not (email and password):
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            headers={"WWW-Authenticate": "Basic"},
+            detail="Missing e-mail or password field"
+        )
+
+    # Authenticate user e-mail and password
+    try:
+        await authenticate_user(
+            email=email,
+            password=password
+        )
+    except ValueError as error:
+        logger.info(f"User failed to authenticate: {email}, reason: {error}")
+        raise HTTPException(
+            status_code=status.HTTP_401_UNAUTHORIZED,
+            headers={"WWW-Authenticate": "Basic"},
+            detail=f"{error}"
+        )
+
+    return {"200": {"model": "Successfully logged!"}}
```

Comment:
Why do we have such a weird structure here? Can't it simply be something like:

```json
{
  "message": "Login successfully!"
}
```

Reply:
That's the format of the FastAPI we used in other places. It looks weird because there isn't a separate response model implemented for the User endpoint. I will add that.